index.d.ts

```diff
@@ -34,18 +34,44 @@ declare namespace pug {
   export function compileFileClient(path: string, options?: Options): string;
 
   /**
+   * Compile a Pug template and render it without locals to html string.
+   */
+  export function render(template: string): string;
+  /**
    * Compile a Pug template and render it with locals to html string.
+   * @param {(Options & LocalsObject)} options Pug Options and rendering locals
```

Comment:
Can you please move the `@param` into the one comment above and remove the duplicate type declaration (it's not needed)?